airflow/providers/google/cloud/operators/dataprep.py

```diff
@@ -51,6 +50,68 @@ def __init__(
 
     def execute(self, context: Dict):
         self.log.info("Fetching data for job with id: %d ...", self.job_id)
-        hook = GoogleDataprepHook(dataprep_conn_id="dataprep_conn_id")
+        hook = GoogleDataprepHook(dataprep_conn_id="dataprep_conn_id",)
         response = hook.get_jobs_for_job_group(job_id=self.job_id)
         return response
+
+
+class DataprepGetJobGroupOperator(BaseOperator):
+    """
+    Get the specified job group.
+    A job group is a job that is executed from a specific node in a flow.
+    API documentation https://clouddataprep.com/documentation/api#section/Overview
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:DataprepGetJobGroupOperator`
+
+    :param job_id The ID of the job that will be requests
+    :type job_id: int
+    :param embed Comma-separated list of objects to pull in as part of the response
+    :type embed: string
+    :param include_deleted if set to "true", will include deleted objects
+    :type include_deleted: bool
+    """
+
+    template_fields = ("job_id",)
+
+    @apply_defaults
+    def __init__(
+        self, *, job_id: int, embed: str, include_deleted: bool, **kwargs
+    ) -> None:
+        super().__init__(**kwargs)
+        self.job_id = job_id
+        self.embed = embed
+        self.include_deleted = include_deleted
+
+    def execute(self, context: Dict):
+        self.log.info("Fetching data for job with id: %d ...", self.job_id)
+        hook = GoogleDataprepHook(dataprep_conn_id="dataprep_conn_id")
+        response = hook.get_job_group(job_id=self.job_id, embed="", include_deleted=self.include_deleted)
```

Comment:
embed?

Reply:
Yes. Just like in the documentation.

`embed | string  
Example:  embed=association.otherAssociation,anotherAssociationComma-separated list of objects to pull in as part of the response. See Embedding Resources for more information.
`

Reply:
```suggestion
        response = hook.get_job_group(job_id=self.job_id, embed=self.embed, include_deleted=self.include_deleted)
```

Reply:
Yes... even looks strange. Good catch.

Reply:
Done.